controllers/front/CartController.php

```diff
@@ -513,15 +526,15 @@ protected function processChangeProductInCart()
                     );
                 } elseif (!$update_quantity) {
                     $this->errors[] = $this->trans(
-                        'You already have the maximum quantity available for this product.',
-                        [],
+                        'You already have the maximum quantity available for product %product%.',
```

Comment:
Sorry I just have a question on this one:
I can't order it because after this quantity it will be out of stock or because a maximum order quantity has been defined?

Reply:
If it is the first option, I would suggest: "
If it is the second option I would suggest: "You have reached the maximum order quantity for product %product%."